java/openmldb-common/src/main/java/com/_4paradigm/openmldb/common/codec/FlexibleRowBuilder.java

```diff
@@ -392,6 +392,10 @@ public boolean build() {
         result.put(baseFieldBuf.array());               // Base data type
         result.put(strAddrBuf.array());                 // String addr
         result.put(stringWriter.toByteArray());         // String value
+
+        // reset strAddrBuf
```

Comment:
`strAddBuf `will be reset in `clear` method. it seems that the `result` will put all buffer that may contains dirty data.  An alternative solution is to putting the used data in `strAddBuf`